Octokit/Models/Response/AccountType.cs

```diff
@@ -27,6 +27,12 @@ public enum AccountType
 	/// 			is attributed to placeholder identities called mannequins.
 	/// </summary>
 	[Parameter(Value = "Mannequin")]
-	Mannequin
+	Mannequin,
+
+        /// <summary>
+        /// Enterprise account
+        /// </summary>
+        [Parameter( Value = "Enterprise" )]
+        Enterprise,
```

Comment:
Used 4-space indents like the majority of the file, while contrary to the `Mannequin` entry above